examples/kagome_full/kagome_options.cpp

```diff
@@ -35,7 +38,13 @@ namespace kagome::options {
       ("keystore,k", po::value<std::string>(&keystore_path)->required(),
        "mandatory, keystore file path")
       ("leveldb,l", po::value<std::string>(&leveldb_path)->required(),
-          "mandatory, leveldb directory path");
+          "mandatory, leveldb directory path")
+      ("p2p_port,p", po::value<uint16_t>(&p2p_port)->default_value(30363),
```

Comment:
make constants for default values

Reply:
why?

Reply:
It is convenient to have predefined values named and described. It enhances readability and simplifies making changes.

Reply:
this is used only in one place and from the construction it is clear that this is default value